README.md

```diff
@@ -0,0 +1,17 @@
+# browserify-rsa
```

Comment:
why are we renaming the readme ? it was lowercase to make completion from the comand line easier